tests/tests/test_integration/test_detection.py

```diff
@@ -50,3 +61,36 @@ def test_detection_full():
     assert isclose(
         num_cells_validation, num_cells_test, abs_tol=DETECTION_TOLERANCE
     )
+
+
+def test_callbacks(signal_array, background_array):
```

Comment:
can you add a docstring to describe the purpose of the test besides the obvious?

Reply:
I can do, but it would proably be a more verbose version of "Tests callbacks", so I'm not sure it would add anything beyond what the function name conveys?

Reply:
Open to suggestions though!

Reply:
Well, I guess naively I have no idea really what the test is doing, but I do get your point. Maybe something about expected behaviour/results?